docs/readmes/basics/quick_start_guide.md

```diff
@@ -185,7 +185,7 @@ We have a fabric command set up to do this:
 
 ```bash
 HOST [magma]$ cd lte/gateway
-HOST [magma/lte/gateway]$ fab -f dev_tools.py register_vm
+HOST [magma/lte/gateway]$ fab register-vm
```

Comment:
Why is `-f dev_tools.py` the function `register_vm` is in that file.

Reply:
That flag does not exist anymore. I have imported the functions into the fabfile in the same directory, so they can be called from there without needing to specify a fabfile.